frontend/src/screens/learner/card.tsx

```diff
@@ -73,203 +80,275 @@ export const Tag: React.FC<{ tag?: string }> = ({ tag }) => {
+                        {study.stages.map((stage, index) => {
+                            return <Group justify='center' align='center' pos='relative' key={index}>
+                                <div style={{ width: '1rem', 
+                                    height: '1rem', 
+                                    backgroundColor: colors.white, 
+                                    borderRadius: '50%',
+                                    border: `2px solid ${colors.purple}`,
+                                }}></div>
+                                {stage.completedAt? <IconCheck color={colors.purple} stroke={5} size="0.5rem"
+                                    style={{
+                                        position: 'absolute',
+                                    }}/> : null}
+                            </Group>
+                        })}
+                    </Group>
+                </Group>
+                <Group justify='center' align='center'>
+                    <Group justify='space-between' w="90%" c={colors.gray70}>
+                        {study.stages.map((stage, index) => {
+                            return <Text key={index} size='xs'>Session {index+1}/{study.stages?.length}</Text>
+                        })}
+                    </Group>
+                </Group>
+            </Stack>
+        </Stack>
     )
 }
 
-const CornerRibbon = styled.div({
-    position: 'absolute',
-    inset: '0 auto auto 0',
-    background: colors.purple,
-    transformOrigin: '100% 0',
-    transform: 'translate(-29.3%) rotate(-45deg)',
-    boxShadow: `0 0 0 999px ${colors.purple}`,
-    clipPath: 'inset(0 -100%)',
-    color: colors.white,
-})
-
-const NewStudyFlag: FC<{study: ParticipantStudy}> = ({ study }) => {
-    if (!study.opensAt) return null
-    const isNew = dayjs(study.opensAt).isAfter(dayjs().subtract(7, 'days'))
-    if (!isNew) return null
-    return (
-        <CornerRibbon>
-            <small>New Study</small>
-        </CornerRibbon>
-    )
+const getTotalCompletedSessions = (study: ParticipantStudy) => {
+    if (!study.stages) return 0
+    return study.stages.filter(stage => stage.completedAt).length + 1
 }
 
-const CompleteFlag: React.FC<StudyCardProps> = ({ study }) => {
-    if (!study.completedAt) return null
+const PointsAndDuration: FC<StudyCardProps> = ({ study }) => {
 
     return (
-        <Box gap
-            align="center"
-            padding="default"
-            css={{
-                backgroundColor: colors.green,
-                position: 'absolute',
-                borderBottomLeftRadius: 20,
-                borderTopLeftRadius: 20,
-                right: 0,
-                top: 16,
-            }}
-        >
-            <Icon icon="checkCircle" color='white' />
-            <span>Complete</span>
-        </Box>
+        <Flex className={cx('mt-auto', 'pt-1')} justify='space-between' align='center' w='100%' c={colors.purple}>
```

Comment:
Another <Group> - also try to avoid `cx()` as thats a part of the emotion / styled packages that we want to remove

Reply:
Done